parl/remote/scripts.py

```diff
@@ -87,6 +87,7 @@ def is_log_server_started(ip_address, port):
             if r.status_code == 400:
                 started = True
                 break
+            time.sleep(3)
```

Comment:
Could you please give some notification here? otherwise users may think our program is hung here.